src/library/scanner/libraryscanner.cpp

```diff
@@ -40,10 +40,8 @@ int execCleanupQuery(FwdSqlQuery& query) {
 
 LibraryScanner::LibraryScanner(
         mixxx::DbConnectionPoolPtr pDbConnectionPool,
-        const UserSettingsPointer& pConfig,
-        QObject* parent)
-        : QThread(parent),
```

Comment:
Don't we need a QThread here? 
I think the alternative would be to use moveToThread().

Reply:
QThread is still the base class. Subclassing QThread when running an event loop is discouraged, but I won't repair this brittle design.